web/functions.php

```diff
@@ -26,13 +26,14 @@ function validateParam($name, $pattern, $defaults): string
 }
 
 // Use YouTube Data API and Shields.io to get an image URL
-function getShieldURL($query, $defaults): string 
+function getShieldURL($query, $defaults): string
 {
     // Check for and validate parameters
     $color = validateParam("color", "/^([0-9a-fA-F]{3}|[0-9a-fA-F]{6}|[A-Za-z]+)$/", $defaults);
     $logo = validateParam("logo", "/^[A-Za-z0-9\-]+$/", $defaults);
     $logoColor = validateParam("logoColor", "/^([0-9a-fA-F]{3}|[0-9a-fA-F]{6}|[A-Za-z]+)$/", $defaults);
-    $style = validateParam("style", "/^[A-Za-z\-]+$/", $defaults);
+    $style = validateParam("style", "/^[A-Za-z\-]+$/", $defaults);                                      // added the sytle and format checl here
```

Comment:
The comment on the end isn't necessary and can be removed

Reply:
I removed these comment.